frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -29,83 +40,203 @@ import { nodeStatus } from '../../status/node';
 
 const NodeTableHeader = () => {
   return [
     {
-      title: 'Name',
+      title: nodeColumnInfo.name.title,
+      id: nodeColumnInfo.name.id,
       sortField: 'metadata.name',
       transforms: [sortable],
-      props: { className: tableColumnClasses[0] },
+      props: { className: nodeColumnInfo.name.classes },
     },
     {
-      title: 'Status',
+      title: nodeColumnInfo.status.title,
+      id: nodeColumnInfo.status.id,
       sortFunc: 'nodeReadiness',
       transforms: [sortable],
-      props: { className: tableColumnClasses[1] },
+      props: { className: nodeColumnInfo.status.classes },
     },
     {
-      title: 'Role',
+      title: nodeColumnInfo.role.title,
+      id: nodeColumnInfo.role.id,
       sortFunc: 'nodeRoles',
       transforms: [sortable],
-      props: { className: tableColumnClasses[2] },
+      props: { className: nodeColumnInfo.role.classes },
     },
     {
-      title: 'Pods',
+      title: nodeColumnInfo.pods.title,
+      id: nodeColumnInfo.pods.id,
       sortFunc: 'nodePods',
       transforms: [sortable],
-      props: { className: tableColumnClasses[3] },
+      props: { className: nodeColumnInfo.pods.classes },
     },
     {
-      title: 'Memory',
+      title: nodeColumnInfo.memory.title,
+      id: nodeColumnInfo.memory.id,
       sortFunc: 'nodeMemory',
       transforms: [sortable],
-      props: { className: tableColumnClasses[4] },
+      props: { className: nodeColumnInfo.memory.classes },
     },
     {
-      title: 'CPU',
+      title: nodeColumnInfo.cpu.title,
+      id: nodeColumnInfo.cpu.id,
       sortFunc: 'nodeCPU',
       transforms: [sortable],
-      props: { className: tableColumnClasses[5] },
+      props: { className: nodeColumnInfo.cpu.classes },
     },
     {
-      title: 'Filesystem',
+      title: nodeColumnInfo.filesystem.title,
+      id: nodeColumnInfo.filesystem.id,
       sortFunc: 'nodeFS',
       transforms: [sortable],
-      props: { className: tableColumnClasses[6] },
+      props: { className: nodeColumnInfo.filesystem.classes },
     },
     {
-      title: 'Created',
+      title: nodeColumnInfo.created.title,
+      id: nodeColumnInfo.created.id,
       sortField: 'metadata.creationTimestamp',
       transforms: [sortable],
-      props: { className: tableColumnClasses[7] },
+      props: { className: nodeColumnInfo.created.classes },
+    },
+    {
+      title: nodeColumnInfo.instancetype.title,
+      id: nodeColumnInfo.instancetype.id,
+      sortFunc: 'nodeIT',
+      transforms: [sortable],
+      props: { className: nodeColumnInfo.instancetype.classes },
+    },
+    {
+      title: nodeColumnInfo.machine.title,
+      id: nodeColumnInfo.machine.id,
+      sortFunc: 'nodeMachine',
+      transforms: [sortable],
+      props: { className: nodeColumnInfo.machine.classes },
+      additional: true,
+    },
+    {
+      title: nodeColumnInfo.version.title,
+      id: nodeColumnInfo.version.id,
+      sortField: 'apiVersion',
```

Comment:
API version? This will always be `v1` and isn't interesting.

I suspect you want OpenShift version, but we don't have that available in the node data. I would take this out.

Reply:
Done.